Client/Frontend/Settings/SettingsNavigationController.swift

```diff
@@ -18,6 +18,18 @@ class SettingsNavigationController: UINavigationController {
     override var preferredStatusBarStyle: UIStatusBarStyle {
         return .default
     }
+    
+    override var supportedInterfaceOrientations : UIInterfaceOrientationMask {
```

Comment:
nit: Space between `:` (on all 3 methods), I think this will cause warnings in SwiftLint?

Reply:
You are right, it didn't show any warning to me :( fixing